central/sensor/service/pipeline/imageintegrations/pipeline.go

```diff
@@ -189,8 +191,17 @@ func setUpIntegrationParams(ctx context.Context, imageIntegration *storage.Image
 		} else {
 			dockerCfg.Insecure = !validTLS
 		}
-		dockerCfg.Endpoint = parseEndpointForURL(dockerCfg.GetEndpoint())
+
+		origEndpoint := dockerCfg.GetEndpoint()
+		dockerCfg.Endpoint = parseEndpointForURL(origEndpoint)
 		description = dockerCfg.GetEndpoint()
+
+		// Ensure the endpoint path is maintained in the description for sourced integrations so
+		// that endpoints with paths (ie: quay.io/rhacs-eng) do not overwrite each other.
```

Comment:
```suggestion
		// that endpoints with paths (ie: quay.io/stackrox-io) do not overwrite each other.
```

I figure we should reference the public one :)